mopro-core/src/middleware/circom/serialization.rs

```diff
@@ -37,16 +39,22 @@ pub fn deserialize_proving_key(data: Vec<u8>) -> SerializableProvingKey {
         .expect("Deserialization failed")
 }
 
-pub fn serialize_inputs(inputs: &SerializableInputs) -> Vec<u8> {
-    let mut serialized_data = Vec::new();
-    inputs
-        .serialize_uncompressed(&mut serialized_data)
-        .expect("Serialization failed");
+pub fn serialize_inputs(inputs: &SerializableInputs) -> Vec<String> {
+    let serialized_bigint_data: Vec<BigUint> =
+        inputs.clone().0.into_iter().map(|x| x.into()).collect();
+    let serialized_data = serialized_bigint_data
+        .into_iter()
+        .map(|x| x.to_string())
+        .collect();
     serialized_data
 }
 
-pub fn deserialize_inputs(data: Vec<u8>) -> SerializableInputs {
-    SerializableInputs::deserialize_uncompressed(&mut &data[..]).expect("Deserialization failed")
+pub fn deserialize_inputs(data: Vec<String>) -> SerializableInputs {
+    let deserialized_data = data
```

Comment:
Ditto this - can we re-use existing serialization methods? don't see anything immediate that'd disqualify strings

Also think for both functions we can actually just add another function because signature is different, so don't need to "overwrite" existing ones in case we want it for other fields